### ISSUE TITLE
qwen returns empty string

### ISSUE THREAD

**Issue by flatsiedatsie** [ngxson/wllama#11]
I noticed something interesting where this tiny model returns an empty string whenever I query it:

https://huggingface.co/Qwen/Qwen1.5-0.5B-Chat-GGUF/resolve/main/qwen1_5-0_5b-chat-q4_0.gguf

**Comment by felladrin** [comment]
Hey, @flatsiedatsie, could it be related to the prompt you're using?

I tried it and got a good response.

Here's the prompt I used:
```
<|im_start|>user
Explain quantum computing like I'm five<|im_end|>
<|im_start|>assistant
```

Could you try again with this one?

---

Console info, for reference:
<img width="457" alt="image" src="https://github.com/ngxson/wllama/assets/418083/f0695d2a-db74-4b6c-9762-2b2c4a3e5cb9">

**Comment by ngxson** [comment]
@felladrin Thank you for having a look. I didn't have time to look into details, but seems like Qwen models are quite sensitive to chat templates (due to their small size - there is no room for errors)

Please let me know if that works for you @flatsiedatsie

**Comment by flatsiedatsie** [comment]
Thanks for testing on your end.

I managed to get output once, but only once. 

I'm using the [tokenizer from Transformers.js](https://github.com/xenova/transformers.js/blob/8bb8c5a33c39aaf33eca286c0c271ed60a94e0da/src/tokenizers.js) to generate the prompts. There was an issue with that, but it was fixed a while ago [as far as I can tell](https://github.com/xenova/transformers.js/issues/572). This process uses Jinja2 templates which are stored on HuggingFace.
